File: beginner/__1000__/1010/1010.py
code1, qnt1, valor1 = input().split()
code2, qnt2, valor2 = input().split()

valor = (int(qnt1) * float(valor1)) + (int(qnt2) * float(valor2))

print(f"VALOR A PAGAR: R$ {valor:.2f}")

File: beginner/__1000__/1010/1010.lua
 
-- (%d+)          -- inteiro positivo
-- ([%-]?%d+)     -- inteiro com sinal opcional
-- (%S+)          -- qualquer token separado por espaço
-- (%a+)          -- palavra

local total = 0

while true do
    local input = io.read()
    if input == nil then
        break
    end

    local _, quantity, price = input:match("(%d+) (%d+) ([%d%.]+)")

    quantity = tonumber(quantity)
    price = tonumber(price)

    total = total + (price * quantity)
end

print("VALOR A PAGAR: R$ " .. string.format("%.2f", total))